runtime/commonMain/src/kotlinx/serialization/modules/SerialModuleBuilders.kt

```diff
@@ -23,6 +23,13 @@ public fun <T : Any> serializersModuleOf(kClass: KClass<T>, serializer: KSeriali
 public inline fun <reified T : Any> serializersModule(serializer: KSerializer<T>): SerialModule =
     serializersModuleOf(T::class, serializer)
 
+/**
+ * Shortcut for [serializersModuleOf] function with type parameter.
+ */
+@ImplicitReflectionSerializer
+public inline fun <reified T : Any> serializersModule(): SerialModule =
```

Comment:
I'm not sure this is useful. Why would one want to use a module of a single default serializer? They wouldn't choose `@ContextualSerialization` in the first place.

Reply:
For the same reason we have `serializersModule(serializer: KSerializer<T>)` and `serializersModuleOf(kClass: KClass<T>, serializer: KSerializer<T>)`

Reply:
Functions you've mentioned have explicit `serializer` parameter, i.e. user-defined one

Reply:
Good point, I'll remove this one.
Do you have any other comments regarding this PR or I can merge it?